src/uwtools/tests/drivers/test_cdeps.py

```diff
@@ -87,7 +87,7 @@ def test_CDEPS_streams(driverobj, group):
     template_file = driverobj._rundir.parent / "template.jinja2"
     with open(template_file, "w", encoding="utf-8") as f:
         print(dedent(template).strip(), file=f)
-    driverobj._driver_config[f"{group}_streams"]["template_file"] = template_file
+    driverobj._config[f"{group}_streams"]["template_file"] = template_file
```

Comment:
Note that unit tests that need to modify a driver's config after it is instantiated should do so via the read-write `._config` / `._config_full` attributes, not the read-only `.config` / `.config_full` properties.